R-packages/evalcast/R/plot_width.R

```diff
@@ -30,18 +34,16 @@ plot_width <- function(cards, alpha = 0.2) {
     select(.data$forecaster, .data$location, .data$forecast_date, .data$coverage) %>%
     unnest(.data$coverage) %>%
     group_by(.data$forecaster, .data$nominal, .data$forecast_date) %>%
-    summarize(median = stats::median(.data$width),
-              max = max(.data$width)) %>%
-    pivot_longer(cols = .data$median:.data$max,
-                 names_to = "summary",
-                 values_to = "width") %>%
+    summarize(level = as.factor(levels),
+              width = quantile(.data$width, probs = levels)) %>%
     ggplot(aes(x = .data$nominal,
                y = .data$width,
                color = .data$forecaster,
-               lty = summary)) +
+               lty = .data$level)) +
     geom_line() +
     facet_wrap(~ forecast_date) +
-    labs(x = "Nominal coverage", y = "Interval width")
+    labs(x = "Nominal coverage", y = "Interval width") + 
+    theme_bw() + theme(legend.position = legend.position)
```

Comment:
should add theme to line with @importFrom ggplot2 so that R CMD check doesn't complain

Reply:
Ok sure.